DigitalLearningSolutions.Web/Helpers/FilteringHelper.cs

```diff
@@ -9,13 +9,34 @@
     public static class FilteringHelper
     {
         public const char Separator = '|';
+        public const char FilterSeparator = '╡';
+
+        public static string? AddNewFilterToFilterBy(string? filterBy, string? filterValue)
+        {
+            if (filterValue != null)
+            {
+                if (filterBy != null)
+                {
+                    if (!filterBy.Contains(filterValue))
+                    {
+                        filterBy = filterBy + FilteringHelper.FilterSeparator + filterValue;

```

Comment:
Should this be FilteringHelper.Separator rather than FilteringHelper.FilterSeparator? I think specifying FilteringHelper is redundant as well.

Reply:
Nope, we have a double separated list. FilteringHelper.Separator is the separator for within the filters i.e. Role|IsCentreAdmin|true. FilteringHelper.FilterSeparator is the separator for between each filters i.e. Role|IsCentreAdmin|true╡Role|IsCmsManager|true

Reply:
Nevermind on the Separator differences, think I initially misunderstood what it was doing.

Reply:
Maybe they need renaming...